build-native.cmd

```diff
@@ -0,0 +1,78 @@
+setlocal
+
+set VS2019=C:\Program Files (x86)\Microsoft Visual Studio\2019
+set VS=
+
+if exist "%VS2019%\Community\" set VS=%VS2019%\Community
+if exist "%VS2019%\Enterprise\" set VS=%VS2019%\Enterprise
+if "%VS%" == "" (
+    echo Can't find VS2019 install
+    exit /b 1
+)
+
+call "%VS%/VC/Auxiliary/Build/vcvarsall.bat" amd64 || exit /b 1
+set BUIDROOT=%cd%
+
+:getopts
+if "%2"=="" (
+	set configuration="Release"
+) else (
+	set configuration=%2%
+)
+
+
+if "%1"=="" (goto :default) else (goto :%1)
+goto :exit
+
+:default
+goto :build
+
+:clean
+echo ### Cleaning the %configuration% build directory
+cd %BUIDROOT%\native
+msbuild winp.vcxproj /t:Clean /p:Configuration=%configuration% /verbosity:minimal /nologo /p:Platform="Win32"
+if %errorlevel% neq 0 exit /b %errorlevel%
+msbuild winp.vcxproj /t:Clean /p:Configuration=%configuration% /verbosity:minimal /nologo /p:Platform="x64"
+if %errorlevel% neq 0 exit /b %errorlevel%
+msbuild sendctrlc\sendctrlc.vcxproj /t:Clean /p:Configuration=%configuration% /verbosity:minimal /nologo /p:Platform="Win32"
+if %errorlevel% neq 0 exit /b %errorlevel%
+msbuild sendctrlc\sendctrlc.vcxproj /t:Clean /p:Configuration=%configuration% /verbosity:minimal /nologo /p:Platform="x64"
+if %errorlevel% neq 0 exit /b %errorlevel%
+msbuild ..\native_test\testapp\testapp.vcxproj /t:Clean /p:Configuration=%configuration% /verbosity:minimal /nologo /p:Platform="Win32"
+if %errorlevel% neq 0 exit /b %errorlevel%
+msbuild ..\native_test\testapp\testapp.vcxproj /t:Clean /p:Configuration=%configuration% /verbosity:minimal /nologo /p:Platform="x64"
+if %errorlevel% neq 0 exit /b %errorlevel%
+goto :exit
+
+:build
+echo ### Building the %configuration% configuration
+cd %BUIDROOT%\native
+REM /verbosity:minimal
+msbuild winp.vcxproj /p:Configuration=%configuration% /nologo /p:Platform="Win32"
+if %errorlevel% neq 0 exit /b %errorlevel%
+msbuild winp.vcxproj /p:Configuration=%configuration% /nologo /p:Platform="x64"
+if %errorlevel% neq 0 exit /b %errorlevel%
+msbuild sendctrlc\sendctrlc.vcxproj /p:Configuration=%configuration% /nologo /p:Platform="Win32"
+if %errorlevel% neq 0 exit /b %errorlevel%
+msbuild sendctrlc\sendctrlc.vcxproj /p:Configuration=%configuration% /nologo /p:Platform="x64"
+if %errorlevel% neq 0 exit /b %errorlevel%
+
+echo ### Building test applications
+msbuild ..\native_test\testapp\testapp.vcxproj /verbosity:minimal /p:Configuration=%configuration% /nologo /p:Platform="Win32"
+if %errorlevel% neq 0 exit /b %errorlevel%
+msbuild ..\native_test\testapp\testapp.vcxproj /verbosity:minimal /p:Configuration=%configuration% /nologo /p:Platform="x64"
+if %errorlevel% neq 0 exit /b %errorlevel%
+
+echo ### Updating WinP resource files for the %configuration% build
+cd %BUIDROOT%
+if not exist src\main\resources mkdir src\main\resources
```

Comment:
Build products in a Maven build generally shouldn't go in `src/main/resources` — they should go somewhere in the `target` directory, which is already excluded from `.gitignore`. You should be able to invoke the build from the `process-resources` Maven lifecycle phase, at which point Maven has copied the resources into `target/` but hasn't packaged the JAR yet. Then you can put your extra resources into `target/` and Maven will proceed to package the JAR with them.

Reply:
Fixed this myself.